src/unity/paths/orgs_orgId_users_userId.yml

```diff
@@ -29,8 +29,8 @@ delete:
     '400':
       description: User trying to remove self
       $ref: '../../common/responses/ServerError.yml'
-    '401':
-      description: Unauthorized
+    '403':
+      description: Cannot remove last user
```

Comment:
Isn't it still possible to get a 401 for this endpoint if someone calls it without being authenticated? Should the 403 be additive here, and not a replacement?

Reply:
It should still be possible to get a `401`, thanks.